Tests/test_file_jpeg2k.py

```diff
@@ -110,10 +110,11 @@ def test_prog_res_rt(self):
         self.assert_image_equal(im, test_card)
 
     def test_reduce(self):
-        with Image.open("Tests/images/test-card-lossless.jp2") as im:
-            im.reduce = 2
-            im.load()
-            self.assertEqual(im.size, (160, 120))
+        for var in ["reduce", "load_reduce"]:
+            with Image.open("Tests/images/test-card-lossless.jp2") as im:
+                setattr(im, var, 2)
+                im.load()
+                self.assertEqual(im.size, (160, 120))
```

Comment:
For readability, let's add a new test instead rather than cramming more in, for example:

```python
    def test_load_reduce(self):
        with Image.open("Tests/images/test-card-lossless.jp2") as im:
            im.load_reduce = 2
            im.load()
            self.assertEqual(im.size, (160, 120))
```

Somewhat related:
* https://testing.googleblog.com/2018/06/testing-on-toilet-keep-tests-focused.html
* https://testing.googleblog.com/2014/07/testing-on-toilet-dont-put-logic-in.html

Reply:
Okay, done